js/demo-app/package.json

```diff
@@ -25,9 +36,14 @@
     "webpack-dev-server": "^3.1.6"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.0.0",
     "@webcomponents/custom-elements": "^1.0.9",
     "copy-webpack-plugin": "^4.5.2",
+    "core-js": "^2.5.7",
+    "custom-event-polyfill": "^1.0.6",
```

Comment:
Hat tip to @j2pichon for finding the issue on ie11.  It was missing support for the CustomEvent constructor.  This polyfill takes care of that.

Working in Edge and IE11.  woot!